js/kucoin.js

```diff
@@ -28,7 +28,7 @@ module.exports = class kucoin extends Exchange {
             'has': {
                 'CORS': undefined,
                 'spot': true,
-                'margin': undefined,
+                'margin': true,
```

Comment:
Shouldn't we only set `margin: true` when we have implemented all/almost all margin features?

Reply:
> Shouldn't we only set `margin: true` when we have implemented all/almost all margin features?

Almost all are implemented except this one https://github.com/ccxt/ccxt/pull/14520